### PR TITLE
Fixes an issue in which the back to top text is unclickable, as it is obscured by the footer.

### PR DESCRIPTION
**Pull request by LevBernstein**
Just some line breaks

### CODE REVIEW

**Review by sam-eng — APPROVED** [review]
Looks solid to me, though I think there's probably a neater way we could do this other than adding in `<br>` tags--if you think you have time, do you think you could look into creating a `<div>` or something to act as a spacer? I think you'd have to assign it a certain height (basing it on em might be good)

**Comment by LevBernstein** [comment]
> Looks solid to me, though I think there's probably a neater way we could do this other than adding in `<br>` tags--if you think you have time, do you think you could look into creating a `<div>` or something to act as a spacer? I think you'd have to assign it a certain height (basing it on em might be good)

I think a better solution might be to make the footer element "exclude" things more; give it more padding and a proper background.

**Comment by sam-eng** [comment]
That definitely sounds better--I think making those changes now (if possible) is better than putting it off to later

File: front-end/src/TermsOfService.js [hunk 1]
```diff
@@ -107,8 +107,9 @@ function TermsOfService() {
 		<ul>
 		    <li>By visiting <Link to="/meet-the-team">our contact page.</Link></li>
 		</ul>
+		<br></br>
```
Comment (sam-eng) [inline front-end/src/TermsOfService.js:110]: A small note--the `br` tag can be self closing, so I think this code could look a little neater as `<br />` rather that `<br></br>`. Other than that, looks good, thanks Lev!

**Review by sam-eng — APPROVED** [review]

File: front-end/src/TermsOfService.js (hunk 1 above)
Comment (LevBernstein) [reply front-end/src/TermsOfService.js:110]: I'll make that change quickly

**Pull request merged**